socorro/middleware/middleware_app.py

```diff
@@ -65,6 +65,7 @@
     (r'/releases/(channels|featured|release)/(.*)', 'releases.Releases'),
     (r'/report/(list)/(.*)', 'report.Report'),
     (r'/server_status/(.*)', 'server_status.ServerStatus'),
+    (r'/signature/first_date', 'signature_first_date.SignatureFirstDate'),
```

Comment:
Legacy! There was a time when our URLs looked like this: ``/search/product/Firefox/version/20.2a/...``. Though I realize now that I forgot a trailing slash that every other URL has. I'll add it before merging.